helmfile.d/charts/gatekeeper/constraints/templates/no-cluster-deletion/constraint.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.noClusterDeletion.enable -}}
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sNoClusterDeletion
+metadata:
+  name: elastisys-no-cluster-deletion
+spec:
+  enforcementAction: {{ .Values.noClusterDeletion.enforcementAction }}
+  match:
+    kinds:
+      - apiGroups: ["*"]
+        kinds:
+          - Cluster
+          - OpenStackCluster
```

Suggestion: I guess this also should be here?
```suggestion
          - OpenStackCluster
          - AzureCluster
```

If so, there's some more places it needs to be added